examples/webgl_mesh_batch.html

```diff
@@ -277,15 +278,10 @@
 					const rotationSpeed = mesh.userData.rotationSpeeds[ i ];
 					const id = ids[ i ];
 
-					mesh.getMatrixAt( id, dummy.matrix );
-					dummy.matrix.decompose( dummy.position, dummy.quaternion, dummy.scale );
-					dummy.rotation.set(
-						dummy.rotation.x + rotationSpeed.x,
-						dummy.rotation.y + rotationSpeed.y,
-						dummy.rotation.z + rotationSpeed.z
-					);
-					dummy.updateMatrix();
-					mesh.setMatrixAt( id, dummy.matrix );
+					mesh.getMatrixAt( id, matrix );
+					rotationMatrix.makeRotationFromEuler( rotationSpeed );
```

Comment:
Maybe compute `rotationMatrix` only once and assign it to `userData`

Reply:
Done